run_unittests.py

```diff
@@ -317,6 +318,35 @@ def test_string_templates_substitution(self):
         cmd = ['@OUTPUT@.out', 'ordinary', 'strings']
         self.assertRaises(ME, substfunc, cmd, d)
 
+    def test_need_exe_wrapper_override(self):
+        config = ConfigParser()
+        config['binaries'] = {
+          'c': '\'/usr/bin/gcc\'',
+        }
+        config['host_machine'] = {
+          'system': '\'linux\'',
+          'cpu_family': '\'arm\'',
+          'cpu': '\'armv7\'',
+          'endian': '\'little\'',
+        }
+
+        with tempfile.NamedTemporaryFile(mode='w+') as configfile:
+            config.write(configfile)
+            configfile.flush()
+            detected_value = mesonbuild.environment.CrossBuildInfo(configfile.name).need_exe_wrapper()
+
+        desired_value = not detected_value
+        config['properties'] = {
+          'needs_exe_wrapper': 'true' if desired_value else 'false'
```

Comment:
__[flake8]__
* [E121] continuation line under-indented for hanging indent

<!-- Comment by [SideCI](https://sideci.com) -->